packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -16,17 +16,14 @@ import {
 import { _usePermissions } from '@internal/react-components';
 import React, { useMemo, useRef } from 'react';
 /* @conditional-compile-remove(one-to-n-calling) */
-import { useCallback, useState } from 'react';
+import { useCallback } from 'react';
 /* @conditional-compile-remove(one-to-n-calling) */
 import { AvatarPersonaDataCallback } from '../../common/AvatarPersona';
 import { containerDivStyles } from '../../common/ContainerRectProps';
 /* @conditional-compile-remove(one-to-n-calling) */
 import { useAdapter } from '../adapter/CallAdapterProvider';
 import { CallControls, CallControlsProps } from '../components/CallControls';
-/* @conditional-compile-remove(one-to-n-calling) */
-import { useSelector } from '../hooks/useSelector';
-/* @conditional-compile-remove(one-to-n-calling) */
-import { callStatusSelector } from '../selectors/callStatusSelector';
+import { useSidePaneState } from '../hooks/useSidePaneState';
```

Comment:
Should this be conditionally compiled as well?

Reply:
indeed! thanks

Reply:
This should be conditionally compiled for one-to-n-calling

Reply:
🤦 - updated!

Reply:
(my secret plan next PR is to remove the fact we have two people panes and make callwithchat hook into this so would remove the cc line, but also involves doing controversial things like creating an imperative handle to the call composite to control the pane from the callwithchat composite 👀)